### DISCUSSION TITLE
CSS class on today's date

### DISCUSSION THREAD

**Discussion by pmackay** [liamcain/obsidian-calendar-plugin#281] [Ideas]
Is there a specific CSS class on today's date? If not, could that be added?

I slightly struggle to see which is today's date, it could be a bit more obvious with a circle or square background highlight.

**Comment by liamcain** [comment]
The CSS class is just `.today`

**Comment by TigBuh99** [comment]
May I have some help concerning this? I was advised to add
```
.calendar {
  --color-background-weekend: GrayText;
 }
```
in obsidian.css because I wanted to change the weekend background color and that works fine. However, I would like today's date to have a red background. How do I do this? Please note that I use the Minimal theme and I don't have any technical knowledge, but I can follow instructions!
